### PR TITLE
Frederick cosby

### PR DESCRIPTION
**Pull request by Frederick-Cosby**

### CODE REVIEW

File: BOWRobot-master/src/main/java/frc/robot/Robot.java [hunk 1]
```diff
@@ -66,6 +67,17 @@ public void robotPeriodic()
 
         Color detectedColor = m_colorSensor.getColor();
         System.out.println("The color is " + detectedColor.toString());
+
+        double IR = m_colorSensor.getIR();
```
Comment (PranavNarala1) [inline BOWRobot-master/src/main/java/frc/robot/Robot.java]: Seems okay.

File: BOWRobot-master/src/main/java/frc/robot/Robot.java [hunk 2]
```diff
@@ -66,6 +67,17 @@ public void robotPeriodic()
...
+
+        SmartDashboard.putNumber("Red", detectedColor.red);
+        SmartDashboard.putNumber("Green", detectedColor.green);
```
Comment (AdrianSantos4290) [inline BOWRobot-master/src/main/java/frc/robot/Robot.java]: dont

File: BOWRobot-master/src/main/java/frc/robot/Robot.java [hunk 3]
```diff
@@ -66,6 +67,17 @@ public void robotPeriodic()
...
+        SmartDashboard.putNumber("Blue", detectedColor.blue);
```
Comment (PranavNarala1) [inline BOWRobot-master/src/main/java/frc/robot/Robot.java]: The code seems good.

File: BOWRobot-master/src/main/java/frc/robot/Robot.java [hunk 4]
```diff
@@ -66,6 +67,17 @@ public void robotPeriodic()
...
+        SmartDashboard.putNumber("IR", IR);
+
```
Comment (PranavNarala1) [inline BOWRobot-master/src/main/java/frc/robot/Robot.java]: blah

File: BOWRobot-master/src/main/java/frc/robot/Robot.java [hunk 5]
```diff
@@ -66,6 +67,17 @@ public void robotPeriodic()
 
         Color detectedColor = m_colorSensor.getColor();
         System.out.println("The color is " + detectedColor.toString());
+
+        double IR = m_colorSensor.getIR();
+
+        SmartDashboard.putNumber("Red", detectedColor.red);
+        SmartDashboard.putNumber("Green", detectedColor.green);
+        SmartDashboard.putNumber("Blue", detectedColor.blue);
+        SmartDashboard.putNumber("IR", IR);
```
Comment (AdrianSantos4290) [inline BOWRobot-master/src/main/java/frc/robot/Robot.java]: f

File: BOWRobot-master/src/main/java/frc/robot/Robot.java (hunk 1 above)
Comment (Frederick-Cosby) [reply BOWRobot-master/src/main/java/frc/robot/Robot.java]: does it?

File: BOWRobot-master/src/main/java/frc/robot/Robot.java (hunk 3 above)
Comment (Frederick-Cosby) [reply BOWRobot-master/src/main/java/frc/robot/Robot.java]: Oh really?

File: BOWRobot-master/src/main/java/frc/robot/Robot.java (hunk 4 above)
Comment (Frederick-Cosby) [reply BOWRobot-master/src/main/java/frc/robot/Robot.java]: blob

File: BOWRobot-master/src/main/java/frc/robot/Robot.java (hunk 2 above)
Comment (Frederick-Cosby) [reply BOWRobot-master/src/main/java/frc/robot/Robot.java]: do

File: BOWRobot-master/src/main/java/frc/robot/Robot.java (hunk 5 above)
Comment (Frederick-Cosby) [reply BOWRobot-master/src/main/java/frc/robot/Robot.java]: h

File: BOWRobot-master/src/main/java/frc/robot/Robot.java (hunk 1 above)
Comment (AdrianSantos4290) [reply BOWRobot-master/src/main/java/frc/robot/Robot.java]: no

**Pull request merged**